### PR TITLE
Do not prevent abolute path in container secret

### PR DESCRIPTION
**Pull request by janisz**
Closes: https://jira.d2iq.com/browse/D2IQ-72705

## Things to consider

- [] Integration tests
- [] System tests
- [] Does it need a feature flag?

### CODE REVIEW

**Comment by pierrebeitz** [comment]
please have a look at the "Lint" section at https://github.com/dcos/dcos-ui/pull/4945#issuecomment-706407307 again. we format with prettier.js

please let the commit message state that this is a fix for out generated changelogs. e.g. `fix: i fixed stuff`

**Comment by janisz** [comment]
Fixed it

What I miss is section in readme or contribute.md with information how to setup up project git hooks to lint commits and code on `git commit`

**Comment by pierrebeitz** [comment]
those hooks are meant to be setup via npm's `preinstall`. that in turn seems to call `./scripts/pre-install` which should install those hooks for you. i don't think we have a prettier-check on precommit though.

File: plugins/secrets/validators/SecretsValidators.ts [hunk 1]
```diff
@@ -16,32 +16,18 @@ interface ServiceError {
...
-
-function isRelativeFilePath(path: string) {
-  return !(
-    path[0] === "/" ||
-    path[path.length - 1] === "/" ||
-    path[0] === "\\" ||
-    path[path.length - 1] === "\\"
-  );
-}
+const secretPathDirectoryPath = /\/$/;
```
Comment (pierrebeitz) [inline plugins/secrets/validators/SecretsValidators.ts:19]: not sure whether that's a good thing to check. 
i'm mostly in favour of letting the backend check stuff. but let's try.

**Pull request merged**